src/dwalk/dwalk.c

```diff
@@ -432,6 +434,9 @@ int main(int argc, char** argv)
             case 'p':
                 print = 1;
                 break;
+            case 'n':
+                walk_opts->no_atime = 1;
```

Comment:
In the man page you say `Must be used with --lite option.`. Do you ensure that's true somewhere below?

Reply:
Yes it is managed by the condition `if(walk_opts->use_stat)` in `mfu_flist_walk_paths()`, if the --lite option is not set, we will not use the path which uses the NO_ATIME for opening directories even if the option has been set.

Reply:
I see, thanks.  I personally would be in favor of an error or warning to the user, either in dwalk after the `getopt_long()` loop is finished, or perhaps in `mfu_flist_walk_paths()`, if the user runs without `--lite` and with `--no-atime`.  I can see a new user overlooking the "use with -l" admonishment, or forgetting about it.

I can see arguments either way, though, so if you disagree you can just mark this conversation resolved.